wordpress-mobile/WordPress-iOS
# Site Monitoring: Add site monitoring menu item

Fixes #22458

## Description
Adds a site monitoring menu item to the maintenance section

## Note
- Currently investigating the conditions to show the site monitoring option by comparing it to Calypso - any updates to the logic will be addressed in a future PR (Slack ref: p1706193407099459/1706106120.703839-slack-C06EFHCP0PN)
- The Site Monitoring screen is a skeleton view for now - this will be updated in a future PR

## How to test
- Enable the site monitoring feature flag
- Switch to an atomic site
- Go to the more menu
- Verify the Site Monitoring menu is shown

<img src="https://github.com/wordpress-mobile/WordPress-iOS/assets/6711616/c1ff9852-ab27-4a7a-aeaf-82c49a6e9300" width=200>

## Regression Notes
1. Potential unintended areas of impact
BlogDetailsVC

2. What I did to test those areas of impact (or what existing automated tests I relied on)
n/a

3. What automated tests I added (or what prevented me from doing so)
n/a

PR submission checklist:


UI Changes testing checklist:
